### PR TITLE
Upgrade konnectors

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

**Comment by ptbrowne** [comment]
I see that you ran `prettier` in the commit to Configure Sentry. I will have merge conflicts for my other PR, https://github.com/konnectors/cozy-konnector-ameli/pull/33, could you split the formatting and the functionality into two commits ? I think it will then be easier for me or you ditch the formatting commit when rebasing, and reapply just afterwards.

Otherwise, looks pretty good ! Happy to finally see prettier in the konnectors. Good job for Sentry too !

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "cozy-konnector-ameli",
   "version": "1.0.6",
-  "description": "Cozy v3 Konnector for Ameli",
+  "description": "Cozy Konnector for Ameli",
```
Comment (ptbrowne) [inline package.json]: :+1:

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   "repository": {
     "type": "git",
-    "url": "https://github.com/konnectors/cozy-konnector-ameli.git"
+    "url": "git+https://github.com/konnectors/cozy-konnector-ameli.git"
```
Comment (ptbrowne) [inline package.json]: Why ?

File: .travis.yml [hunk 3]
```diff
@@ -3,7 +3,8 @@ node_js:
...
+    # kosssi GITHUB_TOKEN
+    - secure: "c4Mdc0HhRtDzi4U3ykwPawh7NvfkpDRzhQlBn0tG31YjZQ1Q2/IUHfKB/nQ1e/xMin8lg4UT/S3UxQZv3+9dn87LfrEBDw7HAmTOQi8uay2Nvvgjo3lpj3DA/YDbTXtvrVx529CfRohCYsVSAJy8QVyFnN1XXyHd2jV3P1JtWNastpKVqqVPXvO8O53GXQvLO4W2z738Z4JjzfCxeItGxhjc45kxwHvN//+VXO9qvJ31IsUPjGopReCf6SaATIVa5v3s6aNvbw83t5eB87O28CIhmjB2naG990D8mEZDsdHXCiyTS8l/qyEtZL8lxnWpySL+m0p0LZ31CFynq25fB1ZC1KdQQoxHZxpxriUxR1MTcU8QrCl0dAmm+aEGRh2b/qp0vfKnB21YayxWJkquli4PeD/MBQFTd6CHH8ZWMOvlHF4UR29Hh5A9l1jXbJfSwulbIF8Pv8ThqY/YNjD0qCQ9D19/lsXhghMBGUEsZKgUvdpgZ2YjE6eBwRFFe0bp85UBa1WaX3Uam5cMaKfCB75Vo3cP/m6VyB6QXSzmdb36KmWF8zY4TBIEVVsis4lFXV7ZDQ9Wpr2bUzq13q3YNngzF2/8+dPs4g+aqIRhz9LniHHDLKzd3gM9adXd5Befmj3GFcVvYGI/6JJuwc6nmxMzKHSv/348knebx44O4AQ="
```
Comment (ptbrowne) [inline .travis.yml]: Why doe it need to be changed ?

File: webpack.config.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ module.exports = {
     new CopyPlugin([
       { from: 'manifest.konnector' },
       { from: 'package.json' },
+      { from: 'yarn.lock' },
```
Comment (ptbrowne) [inline webpack.config.js]: With the commit message, I thought you added `yarn build` to the the yarn script "build". OK for me though.

File: .travis.yml (hunk 3 above)
Comment (kosssi) [reply .travis.yml]: J'ai fait un update d'un token github qui impactait seulement ce konnector ;)

File: package.json (hunk 2 above)
Comment (kosssi) [reply package.json]: pour être iso avec les autres connecteurs ;)

**Comment by doubleface** [comment]
Didn't we agree to use eslint-config-cozy-app with prettier like in the cozy-apps ? Example in https://github.com/cozy/cozy-konnector-bank-cragr

File: package.json (hunk 2 above)
Comment (ptbrowne) [reply package.json]: Je vois pas les URLs comme ca ici : https://github.com/cozy/cozy-collect/pull/508/files

**Comment by ptbrowne** [comment]
True ! Good catch @doubleface

File: package.json (hunk 2 above)
Comment (kosssi) [reply package.json]: Pourquoi tu pointes `collect` je comprends pas.

Voici des liens vers d'autres package.json :
- https://github.com/cozy/cozy-konnector-bank-cragr/blob/master/package.json#L8
- https://github.com/konnectors/cozy-konnector-mgen/blob/master/package.json#L7
- https://github.com/konnectors/cozy-konnector-maif/blob/master/package.json#L7
...

**Pull request merged**